packages/fxa-settings/src/components/PageAvatar/index.tsx

```diff
@@ -44,6 +44,7 @@ export const PageAddAvatar = (_: RouteComponentProps) => {
   const { avatar } = useAccount();
   const alertBar = useAlertBar();
   const [saveEnabled, setSaveEnabled] = useState(false);
+  const isMobile = /mobi/i.test(navigator.userAgent);
```

Comment:
Hmm, what about `isMobileDevice` in `lib/utilities`? Let's merge this check into that check or just prefer that check.

Reply:
Thanks! I didn't think to check if we had one already

Reply:
IDK how I feel about the combined approach but, meh 🤷

Reply:
lol, useragent sniffing is an ugly hack regardless, this LGTM